src/util/endpoints.js

```diff
@@ -137,19 +139,31 @@ exports.initializeCache = async (policyOptions, config) => {
  *
  * @returns {string} - Returns the endpoint, throws error if failure occurs
  */
-exports.getEndpoint = async (switchUrl, fsp, endpointType, options = {}, renderOptions = {}) => {
+exports.getEndpoint = async (switchUrl, fsp, endpointType, options = {}, renderOptions = {}, proxyConfig) => {
```

Comment:
`proxyConfig` should come before all optional arguments or we can make it optional by setting a default value.